packages/home-spa/src/styles/common.scss

```diff
@@ -90,7 +90,7 @@ body {
 }
 .band {
   background-color: var(--op-black);
-  background-image: url('/img/left-footer-cubes.svg'), url('/img/right-footer-cubes.svg');
+  background-image: url('../img/left-footer-cubes.svg'), url('../img/right-footer-cubes.svg');
```

Comment:
Do we really need two separate images for footer cubes? You can use a translated version of the same image. Also, since the footer is used in all SPAs shouldn't we move this image to Assets?

Reply:
@sayak-sarkar This is not part of the footer, this is a one off use case. And AFIK translate isn't supported in firefox https://caniuse.com/?search=translate

Reply:
@hybridx How about: `transform: scaleX(-1);`

Reply:
That could work for the end quote. I'm not so sure about the background image.

Reply:
Changed for the quote svg 🙌🏻. Thanks @sayak-sarkar

Reply:
Not for the background though

Reply:
@hybridx check the documentation SPA's footer!

Reply:
@hybridx https://github.com/1-Platform/one-platform/blob/e55c5b6ae97473e2c6657966767650392282b584/packages/documentation-spa/src/css/custom.css#L56

Reply:
Try this bit of CSS:
```.footer.footer--dark {
  background-image: url('/img/op_footer_cubes_illustration_ltr.svg'), url('/img/op_footer_cubes_illustration_rtl.svg');
  background-repeat: no-repeat;
  background-position: left, right;
  background-color: #000000;
}```